### DISCUSSION TITLE
consider using queues to tell workers what pixels to draw?

### DISCUSSION THREAD

**Discussion by michaelshumshum** [rdeepak2002/reddit-place-script-2022#402] [Ideas]
some people are reporting that several workers are drawing on to the same pixels when they have lower thread delays. i think one way to avoid it is to use queues so only one worker gets a specific task. this entirely removes overlapped tasks.

**Comment by thoroc** [comment]
It would be nice, but how much effort would that require vs how much longer is this project is going to be used?
